browser/net/brave_site_hacks_network_delegate_helper.cc

```diff
@@ -56,7 +56,9 @@ const std::string& GetQueryStringTrackers() {
            // https://github.com/brave/brave-browser/issues/18758
            "gbraid", "wbraid",
            // https://github.com/brave/brave-browser/issues/9019
-           "_hsenc", "__hssc", "__hstc", "__hsfp", "hsCtaTracking"}),
+           "_hsenc", "__hssc", "__hstc", "__hsfp", "hsCtaTracking",
+           // https://github.com/brave/brave-browser/issues/20115
+           "mkt_tok", "igshid"}),
```

Comment:
We can't remove `mkt_tok` because it breaks unsubscription links: https://github.com/brave/brave-browser/issues/9018

Reply:
fixed with the new commit